server/store/flow_store.go

```diff
@@ -3,9 +3,10 @@ package store
 import "fmt"
 
 const (
-	UpdateStatusPropertyName    = "update_status"
-	GetConfirmationPropertyName = "get_confirmation"
-	SubscribePropertyName       = "subscribe"
+	UpdateStatusPropertyName               = "update_status"
+	GetConfirmationPropertyName            = "get_confirmation"
+	ReceiveNotificationsWhileOnMeetingName = "receive_notifications"
```

Comment:
Not sure if we can make the `value` shorter while still being clear. Any changes should apply to other similarly named vars of course

```suggestion
	ReceiveNotificationsDuringMeetingName = "receive_notifications_during_meeting"
```

Reply:
I added your suggestion. If you have any other idea to make it shorter but giving the same information, please let me know and I will change it.

Reply:
You've changed the name of the var, but the main thing I was suggesting was to change the value to `receive_notifications_during_meeting`. The value `receive_notifications` is too vague imo and I would assume by default that it is referring to receiving event notifications in general.